README.md

```diff
@@ -1,3 +1,150 @@
-# Bloom UI Seeds
+# Seeds – Component Library for Bloom
 
-This package is the home of the core UI components for the Bloom affordable housing system, meant to be imported from one or more applications that provide the end-user interface.
+This package serves as the home of [Bloom](https://www.exygy.com/housing)'s core UI components, meant to be imported from one or more Bloom applications. Seeds supersedes Bloom's previous "ui-components" package.
```

Comment:
Should we change the wording from "one or more Bloom applications" to something more generic so that it is implied that this can be used outside of Bloom?